DigitalOffice/UserService/Database/Entities/Company.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace UserService.Database.Entities
+{
+    public class Company
+    {
+        [Key]
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public Guid CEO { get; set; }
```

Comment:
I think CEOId should be used here

Reply:
Actually yeah. Since we mark all guids as PropertyId...
Agree.